spec/controllers/api_v1_slides_controller_spec.rb

```diff
@@ -1,13 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::SlidesController, type: :controller do
+  before(:each) { request.env["HTTP_ACCEPT"] = 'application/json' }
+  render_views
   let(:first_user) { create(:first_user) }
+  list_json_keys = ['id', 'user_id', 'name', 'description', 'category_id', 'key', 'extension', 'num_of_pages', 'created_at', 'category_name', 'username', 'thumbnail_url', 'transcript_url', 'tags']
 
   describe 'GET #index' do
     it 'render index' do
+      create_list(:slide, 2)
       get :index, format: 'json'
       expect(response.status).to eq(200)
       expect(response).to render_template :index
+      json = JSON.parse(response.body);
+      json.each do |j|
+        list_json_keys.each do |k|
+          expect(j.has_key?(k)).to eq(true)
```

Comment:
__[rubocop]__
* Style/PreferredHashMethods: Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->